basiclti/basiclti-tool/src/java/org/sakaiproject/blti/tool/LTIAdminTool.java

```diff
@@ -205,6 +434,30 @@ public String buildToolSitePanelContext(VelocityPortlet portlet, Context context
 		context.put("isAdmin",new Boolean(ltiService.isAdmin()) );
 		context.put("getContext",toolManager.getCurrentPlacement().getContext());
 		
+		//put velocity date tool in the context
+		context.put("dateTool", new DateTool());
+		DateFormat df = DateFormat.getDateTimeInstance(DateFormat.SHORT, DateFormat.SHORT, rb.getLocale());
+		if (df instanceof SimpleDateFormat)
+		{
+			SimpleDateFormat sdf = (SimpleDateFormat)df;
+			context.put("dateToolFormat",sdf.toLocalizedPattern());
```

Comment:
Why do you need the `dateToolFormat` in the context at all, when you create the formatter you specify the format and the locale so if you just call `df.format(Date)` it should give you the correct output shouldn't it?